### ISSUE TITLE
App Open Function - Non Urgent

### ISSUE THREAD

**Issue by MrRedBeard** [Kitchen-JS/KitchenWindowsManager#46]
I need a way to handle the opening of an app or URL like your about author "app". 
I need to pass off the action to a Cordova/Electron method/plugin to open the URL outside the app.

**Comment by MrRedBeard** [comment]
Per @ctrachte use Onclick event

**Comment by ctrachte** [comment]
Yes this can be accomplished through the 'onclick' option in the toolbar items object array. Simply pass it an onclick function of your own, or write one inside the options object.
